crates/holochain/CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ default_semver_increment_mode: !pre_minor dev
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/). This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
+- Remove the integration step from the app validation and authored ops workflows. Instead, integration is only handled by the integrate workflow.
+- Add integration of `StoreRecord` and `StoreEntry` ops to integrate workflow.
+- The integrate workflow now integrates **all** valid `RegisterAddLink` ops instead of only ones that link to a `StoreEntry`.
+- Update doc-comment about CreateLink Action
+- Fix issue where genesis actions weren't integrated when others were ready to integrate. When nothing had been integrated yet then we started integration at the value of how many ops were `ready_to_integrate` so if we had other ops that were ready then the ranged started at them instead of at genesis (index 0).
```

Comment:
Fantastic!
```suggestion
- Fix issue where genesis actions weren't integrated when others were ready to integrate. When nothing had been integrated yet then we started integration at the value of how many ops were `ready_to_integrate` so if we had other ops that were ready then the range started at them instead of at genesis (index 0).
```